conf/modules.config

```diff
@@ -47,7 +47,7 @@ process {
         publishDir = [
             path: { "${params.outdir}/fastq" },
             mode: params.publish_dir_mode,
-            pattern: '*.fastq',
+            pattern: '*.merged.fastq.gz',
```

Comment:
Hi @bounlu, given the comment on the following line, I'm guessing the pipeline devs would rather do that not by default

Reply:
Ohh I see, gotcha.

But currently it does not output anything so the fastq folder is empty. It would be nicer to add a boolean as the placeholder rather than wrong pattern. 😉